src/shogun/mathematics/linalg/SGLinalg.h

```diff
@@ -0,0 +1,56 @@
+#include <shogun/mathematics/linalg/LinalgBackendBase.h>
+#include <shogun/mathematics/linalg/LinalgBackendEigen.h>
+
+#ifndef SG_LINALG_H__
+#define SG_LINALG_H__
+
+#ifdef HAVE_CXX11
+
+namespace shogun
+{
+
+class SGLinalg
+{
+public:
+	SGLinalg()
+	{
+		cpu_backend = new LinalgBackendEigen();
+		gpu_backend = NULL;
+	}
+
+	inline void set_cpu_backend(LinalgBackendBase* backend)
```

Comment:
a header defined + declared function is inlined by default... i.e. that `inline` is just redundancy